scripts/validator/index.ts

```diff
@@ -138,6 +154,22 @@ class Validator {
     });
   }
 
+  private generateAccount() {
+    const keyring = new Keyring({ type: "sr25519", ss58Format: 2 });
+    const mnemonic = mnemonicGenerate(MNEMONIC_WORDS_COUNT);
```

Comment:
We should share the mnemonic with a user. Let's do it via console. Also, we should be sure that it was copied by user. Let's use the same mechanism as for Node launch with `y/n`.